src/tinyformat.h

```diff
@@ -898,7 +898,7 @@ class FormatListN : public FormatList
 // Special 0-arg version - MSVC says zero-sized C array in struct is nonstandard
 template<> class FormatListN<0> : public FormatList
 {
-    public: FormatListN() : FormatList(0, 0) {}
+    public: FormatListN() : FormatList(nullptr, 0) {}
```

Comment:
Agree, we should generally minimize changes to tinyformat.h, and if it is just style it just adds burden on us with no clear gain. 

If you get upstream to accept this it's okay with me. But I think they want to keep compatiblity with older C++.